csharp/src/Google.Protobuf.Benchmarks/Benchmarks.cs

```diff
@@ -38,7 +38,11 @@ public static partial class BenchmarksReflection {
 
   }
   #region Messages
-  public sealed partial class BenchmarkDataset : pb::IMessage<BenchmarkDataset> {
+  public sealed partial class BenchmarkDataset : pb::IMessage<BenchmarkDataset>,
```

Comment:
Are there other messages in this repo to update? Should update them all to support IBufferMessage